packages/code-du-travail-ui/src/Tabs/index.js

```diff
@@ -51,41 +52,55 @@ export default Tabs;
 
 const StyledTabs = styled(RootTabs)`
   margin-bottom: ${spacing.large};
+`;
 
-  .react-tabs__tab-list {
-    position: relative;
-    top: 1px;
-    display: flex;
-    margin: 0;
-    padding: 0;
-    list-style-type: none;
+const StyledTabList = styled(TabList)`
+  display: flex;
+  flex-wrap: nowrap;
+  position: relative;
+  top: 1px;
+  margin: 0;
+  padding: 0;
+  max-width: 100%;
+  list-style-type: none;
+  overflow-x: auto;
+  @media (max-width: ${breakpoints.tablet}) {
+    flex-wrap: wrap;
   }
+`;
 
-  .react-tabs__tab {
-    margin-right: ${spacing.tiny};
-    padding: ${spacing.small} ${spacing.base};
-    font-size: ${fonts.sizeH6};
-    background-color: ${colors.darkBackground};
-    border: 1px solid ${colors.elementBorder};
-    border-bottom: 1px solid ${colors.primaryBackground};
-    border-top-left-radius: ${box.lightBorderRadius};
-    border-top-right-radius: ${box.lightBorderRadius};
-    cursor: pointer;
-    &:hover {
-      text-decoration: underline;
-    }
+const StyledTab = styled(Tab)`
+  margin-right: ${spacing.tiny};
+  padding: ${spacing.small} ${spacing.base};
+  font-size: ${fonts.sizeH6};
+  background-color: ${colors.darkBackground};
+  border: 1px solid ${colors.elementBorder};
+  border-bottom: 1px solid ${colors.primaryBackground};
+  border-radius: ${box.lightBorderRadius} ${box.lightBorderRadius} 0 0;
+  cursor: pointer;
+  &:hover {
+    text-decoration: underline;
   }
-
-  .react-tabs__tab--selected {
+  &[aria-selected="true"] {
     color: ${colors.primaryText};
     background-color: ${colors.primaryBackground};
   }
+  @media (max-width: ${breakpoints.tablet}) {
+    flex: 1 1 auto;
+    margin: ${spacing.tiny};
+    border-bottom: 1px solid ${colors.elementBorder};
+    border-radius: ${box.lightBorderRadius};
+  }
+`;
 
-  .react-tabs__tab-panel--selected {
+const StyledTabPanel = styled(TabPanel)`
+  &.react-tabs__tab-panel--selected {
     padding: ${spacing.base};
     border: 1px solid ${colors.primaryBackground};
-    border-top-right-radius: ${box.borderRadius};
-    border-bottom-right-radius: ${box.borderRadius};
-    border-bottom-left-radius: ${box.borderRadius};
+    border-radius: 0 ${box.borderRadius} ${box.borderRadius} ${box.borderRadius};
```

Comment:
why `0` for the top-left border radius ?

Reply:
because the tab on the  far left needs to sit on a neat corner 
<img width="748" alt="Screenshot 2019-06-11 at 14 49 01" src="https://user-images.githubusercontent.com/705453/59273288-1d69e280-8c58-11e9-8f9e-4ac94c06cea0.png">